datatypes/src/primitives/multi_point.rs

```diff
@@ -0,0 +1,46 @@
+use crate::error;
+use crate::primitives::Coordinate2D;
+use crate::util::Result;
+use geojson::Geometry;
+use snafu::ensure;
+
+pub struct MultiPoint<'g> {
+    coordinates: &'g [Coordinate2D],
```

Comment:
to be more precise this field could be named "point_coordinates". Additionally, we should think of a way to remove the lifetime in the future. Maybe we could do something like T: AsRef<[Coordinate2D]>. Or maybe this is a MultiPointRef -_-

Reply:
dc6e723